src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -528,6 +528,82 @@ public function seeJson(array $data = null, $negate = false)
     }
 
     /**
+     * Assert that the response contains the given JSON keys.
+     *
+     * Keys can be formatted with dot notation, or as nested arrays:
+     *   $keys = [ 'key1', 'key2' => ['subkey1'], 'key3.subkey2' ]
+     *
+     * Would match this json reponse:
+     *   {"key1": "data", "key2": {"subkey1": "data"}, "key3": {"subkey2": "data"}}
+     *
+     * @param  array|null  $data A list of JSON keys to look for
```

Comment:
Please don't document parameters inline like this. If extra clarification is needed, please include that in the "long description".